File: c/2-control structures/workout/csw-3.c
#include <stdio.h>
int main()
{
    int a;
    scanf("%d\n",&a);
    if(a>=1&&a<=5)
    {
        printf("5%%\n");
        
    }
    else if(a>=6&&a<=10)
        printf("10%%");
    else if(a>10)
        printf("15%%\n");
    else
        printf("Invalid Input");
    
}

File: python/3-conditional statements/workout/csw-3.py
try:
    year = int(input())

    if year < 0:
        print("Invalid input")
    elif (year % 4 == 0 and year % 100 != 0) or (year % 400 == 0):
        print("Leap Year")
    else:
        print("Not a Leap Year")

except ValueError:
    print("Invalid input")